src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimizeBatched.cpp

```diff
@@ -1649,28 +1655,24 @@ bool QMCFixedSampleLinearOptimizeBatched::one_shift_run()
         invMat(i, i) = bestShift_i * bestShift_s;
     }
 
-    // compute the inverse of the overlap matrix
-    {
-      ScopedTimer local(invert_olvmat_timer_);
-      invert_matrix(invMat, false);
-    }
-
     // apply the overlap shift
     for (int i = 1; i < N; i++)
       for (int j = 1; j < N; j++)
         hamMat(i, j) += bestShift_s * ovlMat(i, j);
 
-    // multiply the shifted hamiltonian matrix by the inverse of the overlap matrix
-    qmcplusplus::MatrixOperators::product(invMat, hamMat, prdMat);
-
-    // transpose the result (why?)
-    for (int i = 0; i < N; i++)
-      for (int j = i + 1; j < N; j++)
-        std::swap(prdMat(i, j), prdMat(j, i));
+    RealType lowestEV;
+    // compute the lowest eigenvalue and the corresponding eigenvector
+    if (ev_solver_ == "general") {
+      app_log() << "  Using generalized eigenvalue solver (ggev)" << std::endl;
+      lowestEV = getLowestEigenvector_Gen(hamMat, invMat, parameterDirections);
```

Comment:
> The code using the generalized eigensolver has been in the code, but has not been connected. 

Is it not connected here?

Reply:
In the past the generalized eigensolver was not connected, but it is connected now.

Reply:
I see what you mean. PR description reflects the code in the branch. So "has not been connected" confused me.

Reply:
I updated the PR description to be more clear.